PatchesAndAreasApi/PatchesAndAreasApi.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="AWSXRayRecorder.Core" Version="2.11.1" />
     <PackageReference Include="AWSXRayRecorder.Handlers.AspNetCore" Version="2.8.1" />
     <PackageReference Include="AWSXRayRecorder.Handlers.AwsSdk" Version="2.9.1" />
+    <PackageReference Include="Hackney.Core.Authorization" Version="1.79.0-feat-regenerate-0002" />
```

Comment:
dont forget to update this nuget version

Reply:
Switched to main version in https://github.com/LBHackney-IT/patches-and-areas-api/pull/49/commits/55696bde22562b051469b5e6470c1611f95e3d83